tests/test_testRepo.py

```diff
@@ -53,7 +53,7 @@ def setUpClass(cls):
         addDataIdValue(cls.creatorButler, "instrument", "notACam")
         addDataIdValue(cls.creatorButler, "instrument", "dummyCam")
         addDataIdValue(cls.creatorButler, "physical_filter", "k2020", band="k", instrument="notACam")
-        addDataIdValue(cls.creatorButler, "physical_filter", "l2019", instrument="dummyCam")
+        addDataIdValue(cls.creatorButler, "physical_filter", "l2019", band="l", instrument="dummyCam")
```

Comment:
I think this line is actually redundant now, since its purpose was to test that you could omit `band`. Does the test suite break if you delete this line?

Reply:
Yes, and it sounds like the preferred approach is to revert this change and try to make the test pass without it (as per the other thread).

Reply:
If the test breaks, then the line is needed either way.